installutils/kubeinstall/kube_installer.go

```diff
@@ -613,6 +614,10 @@ func (r *KubeInstaller) ListAllResources(ctx context.Context) kuberesource.Unstr
 	return r.cache.List()
 }
 
+func (r *KubeInstaller) RefreshResourcesCache(cache *Cache) {
```

Comment:
not totally sure how this is intended to be used -- if updated in the background from a goroutine, maybe we should protect this with a mutex? if updated prior to using the installer, then why not just create a new installer?

Reply:
or by re-calling "Init" on the cache object? 

Not really opposed to this change, just not sure i can see the full picture / can't reason about the tradeoffs of this vs. alternative approach